docs/modules/ROOT/pages/cops_style.adoc

```diff
@@ -5777,6 +5777,9 @@ a = if cond
 else
   c
 end
+return cond ?
```

Comment:
I guess we shouldn't auto-correct this, but I think we should still flag this, as it looks quite weird.

Reply:
Ah, I get it and updated this PR!